core/templates/css/oppia.css

```diff
@@ -4119,7 +4119,8 @@ oppia-noninteractive-link {
 }
 
 .oppia-about-otter-speech-bubble h3 {
-  color: #000;
+  color: #333333;
```

Comment:
Can you use `#333` instead? When possible we prefer to use the 3 char version for HEX codes (see [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#css)).

Reply:
That's right. Will do the needful and recommit.